src/net.cpp

```diff
@@ -1234,6 +1234,8 @@ void MapPort()
 // The second name should resolve to a list of seed addresses.
 static const char *strDNSSeed[][2] = {
     {"seed.lkrcoin.io", "seed.lkrcoin.io"},
+    {"seed2.lkrcoin.io", "seed2.lkrcoin.io"},
+    {"explorer.lkrcoin.io", "explorer.lkrcoin.io"}
```

Comment:
dig -t NS explorer.lkrcoin.io , dig -t NS seed2.lkrcoin.io and dig -t NS seed.lkrcoin.io 

not working :(